src/bin/nickel.rs

```diff
@@ -81,11 +81,15 @@ enum Command {
     /// Generates the documentation files for the specified nickel file
     #[cfg(feature = "doc")]
     Doc {
-        /// Specify the path of the generated documentation file. Default to
+        /// The path of the generated documentation file. Default to
         /// `~/.nickel/doc/<input-file>.md` for input `<input-file>.ncl`, or to
         /// `~/.nickel/doc/out.md` if the input is read from stdin.
         #[structopt(short = "o", long, parse(from_os_str))]
         output: Option<PathBuf>,
+        /// The output format for the generated documentation. Defaults
+        /// to rendered markdown. Possible values: markdown, json
+        #[structopt(long)]
```

Comment:
Setting a default value with structopt will print it automatically (see the help on any subcommand, and the documentation of --color for an example).

```suggestion
        /// The output format for the generated documentation. Possible values: markdown, json
        #[structopt(long, default_value = "markdown")]
```